spec/list-actions.lsl

```diff
@@ -35,4 +35,8 @@ define action LIST_RESIZE(
 define action LIST_DUP(
         aList: list<any>
     ): list<any>;
-
+    
+define action LIST_REMOVE(
+        aList: list<any>,
+        index: int
```

Comment:
I suggest adding `count` parameter to extend functionality of this action on cases when a continuous group of multiple items being removed at the same time.

Reply:
Was added count param.